.gitignore

```diff
@@ -7,6 +7,7 @@ _*.sh
 vendor
 glide.lock
 *.DS_Store
+go.sum
```

Comment:
@tsconn23 what is the current guidance on go.sum committing? I thought that we had started to commit the go.sum and update it when the dependencies change

Reply:
@anonymouse64 @tsconn23 I also believe we should commit the go.sum but it saw in edgex-go repo that it wasn't so I followed this model... Thought it was maybe an already discussed decision.

Reply:
FYI https://github.com/golang/go/wiki/Modules#releasing-modules-all-versions

Reply:
ah actually it was discussed in one of the devops calls : https://github.com/edgexfoundry/edgex-go/issues/1141

Reply:
I agree with you I would like to know the rationale behind...

Reply:
@anonymouse64 Ian, shall we (for the time being) follow the devops group recommendation (even if I'm not convinced) and complete/close that PR. Until we get more insight or a change of practice, committing the go.sum in a next future will not require much effort anyways.